src/aktualizr_info/main.cc

```diff
@@ -32,7 +32,13 @@ int main(int argc, char **argv) {
       exit(EXIT_SUCCESS);
     }
 
-    std::string sota_config_file = vm["config"].as<std::string>();
+    std::string sota_config_file = "/usr/lib/sota/sota.toml";
+    if (vm.count("config") != 0) {
+      sota_config_file = vm["config"].as<std::string>();
+    } else if (boost::filesystem::exists("/tmp/aktualizr_config_path")) {
+      sota_config_file = Utils::readFile("/tmp/aktualizr_config_path");
```

Comment:
What's the deal with this temp file?

Reply:
Oh, I see, that's the way to figure out whatever config aktualizr is using. Seems alright.